tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -26,6 +48,92 @@ class TestPVCCreationPerformance(E2ETest):
 
     pvc_size = "1Gi"
 
+    def get_rbd_or_fs_image_count(self, pool_name, interface):
+        """
+        This function return the number of rbd images or fs sub volumes
+        created in the backend.
+
+        Args:
+            pool_name (str): the name of the pool to look into
+            interface (str): the name of the interface (block / filesystem)
```

Comment:
@Avilir Interface param is misleading. I copied this function and performed some testing. Based on param name interface name I provided `constants.CEPHFS_INTERFACE` value and the test failed. Later checked the value in this PR jenkins validation job logs, observed we are using `constants.CEPHFILESYSTEM` which is resource/kind https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L92

Reply:
i renamed the name of the argument